features/scripts/launch_ios_simulators.sh

```diff
@@ -1,10 +1,18 @@
 #!/usr/bin/env bash
 
 INSTALL_PATH=build/Build/Products/Debug-iphonesimulator/iOSTestApp.app
-OS_VERSION=${MAZE_SDK:="11.2"}
+XCODE_VERSION=$(xcodebuild -version)
```

Comment:
if I may propose a change to this file: https://github.com/bugsnag/bugsnag-cocoa/pull/362
this is PR to this branch
Runtime selection using full path `com.apple.CoreSimulator.SimRuntime.iOS-12_1` works for both Xcode 10.1 and 10.2.1, so we do not need to check for Xcode version. And selecting simulator by short name such as "iPhone 8" works too.